BookMiPlace/BookMiPlaceRails
# Use UUIDs as primary keys - not integers

#### What's this PR do?
Sets up ActiveRecord to use UUIDs, rather than integers, as all records' primary keys.

##### Background context
Using UUIDs, rather than integers, as primary keys offers a number of security, and other, advantages.

#### How should this be manually tested?
Create a new model/ table and check that it's primary key is a UUID, not an integer.

#### Screenshots

![image](https://user-images.githubusercontent.com/1453680/44191218-2e25f680-a122-11e8-9185-71761f931a1c.png)

produces...

![image](https://user-images.githubusercontent.com/1453680/44191201-21090780-a122-11e8-9122-3a82512b246e.png)


---

#### Migrations
Yes - one. Will need to run:
`rails db:migrate`
in the terminal.
